plugins/extractors/mysql/mysql.go

```diff
@@ -31,31 +31,10 @@ var defaultDBList = []string{
 	"sys",
 }
 
-// Config holds the connection URL for the extractor
-type Config struct {
-	ConnectionURL string `mapstructure:"connection_url" validate:"required"`
-}
-
 var sampleConfig = `connection_url: "admin:pass123@tcp(localhost:3306)/"`
 
-// Extractor manages the extraction of data from MySQL
-type Extractor struct {
-	excludedDbs map[string]bool
-	logger      log.Logger
-	config      Config
-	db          *sql.DB
-	emit        plugins.Emit
-}
-
```

Comment:
Same is true for Extractor struct, wanted to move it to `utils/sql_db.go` so that it is not needed to be declared across all the extractors.

But for that I will also need to declare the fields as public, i.e, logger -> Logger.